llama.cpp

```diff
@@ -1814,13 +1818,26 @@ struct llama_tokenizer {
         work_queue_.push(bigram);
     }
 
+    bool skip_merge()
+    {
+        std::uniform_real_distribution<> gen(0.0, 1.0);
+        if (dropout_ <= 0.0) {
+            return false;
+        }
+        if (dropout_ >= 1.0)
```

Comment:
please add block brackets

Reply:
also, why would you specify > 100% dropout?

Reply:
i believe it is used for input validation and protection. Copied from https://github.com/google/sentencepiece/blob/master/src/bpe_model.cc#L132-L139